pydy/tests/test_models.py

```diff
@@ -19,8 +16,8 @@ def test_multi_mass_spring_damper():
 
     sys = multi_mass_spring_damper()
 
-    assert Counter(sys.constants_symbols) == Counter([k0, c0, m0])
-    assert sys.specifieds_symbols == list()
+    assert sys.constants_symbols == {k0, c0, m0}
```

Comment:
```pydy.tests.test_models.test_multi_mass_spring_damper_double``` would sometimes pass, the order of the variables would change between tests. The other 4 test with failures were failing due to the order changing.

I believe the order is fixed once they arrive to System but I need to look over it again. I'm going to push a new patch set with all changes that you've pointed out _except_ the set thing.